### PR TITLE
added cards

### PR DESCRIPTION
**Pull request by shaksham08**
Added Card components

Cards with badges
Cards with text overlay
Text only cards
Vertical/Horizontal cards
Cards with shadow

image
image

### CODE REVIEW

File: components/cards/index.html [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+          <div class="card__content">
+            <div class="card card__vertical">
+              <img class="card__img" src="../../assets/camera.jpg" alt="img" />
+              <div class="card__details">
+                <h3 class="card__title">Canon D6500</h3>
+                <small class="card__sub-title"
+                  >Canon D6500 super zoom camera</small
+                >
+              </div>
+              <div class="card__price-section">
```
Comment (t007rushi) [inline components/cards/index.html]: you can follow one convention

**Review by t007rushi — APPROVED** [review]
do required changes else good

File: components/cards/style.css [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+  padding: 2rem;
+  margin-top: 2rem;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.3rem;
+  box-shadow: 5rem 5rem 5rem 5rem;
+  box-shadow: 0px 0px 5px 0px var(--gray-light-primary);
```
Comment (harshitadaswani) [inline components/cards/style.css]: Why two same properties in the same class?

**Review by harshitadaswani — APPROVED** [review]
Just have a look at the comment if you have missed the point. Otherwise, you can merge it

File: components/cards/style.css (hunk 2 above)
Comment (shaksham08) [reply components/cards/style.css]: will remove it

**Pull request merged**